examples/default/main.c

```diff
@@ -47,6 +52,14 @@ int main(void)
     rtc_init();
 #endif
 
+#ifdef MODULE_GNRC
```

Comment:
Hm, true, but in fact that's not what I really want here. I want to check if the current platform has gnrc support. Looking at the dependencies we could check for `MODULE_GNRC_NOMAC` here.